docs/source/user-guide/concepts/installing-with-conda.rst

```diff
@@ -0,0 +1,35 @@
+=====================
+Installing with conda
+=====================
+
+.. image:: /img/installing-with-conda.png
+    :align: right
+
+.. _installing-with-conda:
+
+
+To install conda packages, in the terminal or an Anaconda Prompt, run::
+
+  conda install [packagename]
+
+
+During the install process, files are extracted into the
+``install`` prefix, except for files in the ``info/``
```

Comment:
>"During the install process, files are extracted into the``install`` prefix, except for files in the ``info/`` directory"
I think this is confusing to me. I thought files are extracted into the specified environment, defaulting the current environment?

Reply:
eg. This can be changed to something like "During the install process the files are extracted into the specified environment, defaulting to the current environment if none is specified"